src/injector.js

```diff
@@ -159,6 +119,47 @@ export default class Injector {
     }
   }
 
+  injectIntoDOM(dependencies, idx = 0, type = null) {
+    if (idx >= dependencies.length) { return; }
+
+    // inject order: explicitly provided < cached in local storage < printed
+    // raw only as fallback if printed fails
+    type = (dependencies[idx][type] && type) || (dependencies[idx]['cached'] && 'cached') || (dependencies[idx]['printed'] && 'printed');
```

Comment:
Ah damn me. It's the `'printed'`, `'unprinted'` or `'cached'` you wanted to return in order to assign it to `type`. Gotcha, didn't get that intend while looking at the line. Nifty.